docs/resource/README.md

```diff
@@ -219,9 +193,9 @@ All custom identifiers SHOULD be stable across different versions of an implemen
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
-## Telemetry Distribution (Experimental)
+## Telemetry SDK
```

Comment:
this one should probably be 

```suggestion
## Telemetry Distribution
```

there is another "Telemetry SDK" on line 145